game/scripts/npc/abilities/siltbreaker/siltbreaker_boss_protection.txt

```diff
@@ -0,0 +1,113 @@
+        "Target"                                          "CASTER"
+      }
+    }
+
+    "OnSpellStart"
+    {
+      "ApplyModifier"
+      {
+        "ModifierName"                                    "modifier_siltbreaker_boss_protection"
+        "Target"                                          "CASTER"
+        "Duration"                                        "%duration"
+      }
+      "RemoveModifier"
+      {
+        "ModifierName"                                    "modifier_siltbreaker_boss_protection_trigger"
+        "Target"                                          "CASTER"
+      }
+    }
+
+    "Modifiers"
+    {
+      "modifier_siltbreaker_boss_protection_trigger"
+      {
+        "Passive"                                         "1"
+        "IsHidden"                                        "1"
+        "IsDebuff"                                        "0"
+        "IsPurgable"                                      "0"
+        "IsBuff"                                          "1"
+        "IsStunDebuff"                                    "0"
+
+        "OnStateChanged"
+        {
+          "RunScript"
+          {
+            "ScriptFile"                                  "abilities/siltbreaker/siltbreaker_protection.lua"
+            "Function"                                    "siltbreaker_protection_trigger"
+          }
+        }
+      }
+
+      "modifier_siltbreaker_boss_protection"
+      {
+        "Passive"                                         "1"
+        "IsHidden"                                        "0"
+        "IsDebuff"                                        "0"
+        "IsPurgable"                                      "0"
+        "IsBuff"                                          "1"
+        "IsStunDebuff"                                    "0"
+        "Priority"                                        "MODIFIER_PRIORITY_SUPER_ULTRA"
+
+        "Duration"                                        "%duration"
+        // If state changes, reapply the modifier
+        "OnStateChanged"
+        {
+          "RunScript"
+          {
+            "ScriptFile"                                  "abilities/siltbreaker/siltbreaker_protection.lua"
+            "Function"                                    "siltbreaker_protection_purge"
+          }
+        }
+
+        "OnDestroy"
+        {
+          "ApplyModifier"
+          {
+            "ModifierName"                                    "modifier_siltbreaker_boss_protection_trigger"
+            "Target"                                          "CASTER"
+          }
+        }
+
+        "States"
+        {
+          "MODIFIER_STATE_SILENCED"  "MODIFIER_STATE_VALUE_DISABLED"
+          "MODIFIER_STATE_MUTED"  "MODIFIER_STATE_VALUE_DISABLED"
+          "MODIFIER_STATE_STUNNED"  "MODIFIER_STATE_VALUE_DISABLED"
+          "MODIFIER_STATE_HEXED"  "MODIFIER_STATE_VALUE_DISABLED"
```

Comment:
Why does it only trigger on stun or silence but block Hex and Mute as well?

Reply:
I'll remove them. It is there because I thought block them in the beginning bit changed my mind.
The main drive for this change is to force heros with bash to follow the mechanics and nota hope to Nash the boss before it deliver his attacks